core/src/retransmit_stage.rs

```diff
@@ -428,7 +428,9 @@ fn retransmit(
         // neighborhood), then we expect that the packet arrives at tvu socket
         // as opposed to tvu-forwards. If this is not the case, then the
         // turbine broadcast/retransmit tree is mismatched across nodes.
-        if packet.meta.forward == (my_index % DATA_PLANE_FANOUT == 0) {
+        let anchor_node = my_index % DATA_PLANE_FANOUT == 0;
```

Comment:
nit: `anchor_node` -> `is_on_critical_path`

Reply:
I am using the same terminology as in here:
https://github.com/solana-labs/solana/blob/c2191d885/gossip/src/cluster_info.rs#L2857